prisma/migrations/20220426193548_create_deposit/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "Deposit" (
```

Comment:
Can you regenerate this migration file with the new mapped name?

Reply:
done